### ISSUE TITLE
Erroneous comments in documentation HTML files

### ISSUE THREAD

**Issue by vlakoff** [GyanD/codexffmpeg#160]
Looking at source code of `doc/*.html` files, in a ffmpeg build from 2022 there were comments:

```
<!-- Created by GNU Texinfo 6.8, https://www.gnu.org/software/texinfo/ -->
```

but in the ffmpeg build that I downloaded now, these comments have become:
```
<!-- Created by , GNU Texinfo 7.1.1 -->
```

Notice the first "empty" field.

**Comment by GyanD** [comment]
That looks to be a texinfo bug.

**Comment by vlakoff** [comment]
I've had a look at Texinfo code (I used [this mirror](https://github.com/debian-tex/texinfo)), but could not figure out the cause for this.
Sadly they don't have a GitHub page, only a [mailing list](https://www.gnu.org/software/texinfo/)…

I noticed BtbN builds have the same problem (Texinfo 7.1).

**Comment by GyanD** [comment]
Has to be fixed upstream.

**Comment by vlakoff** [comment]
Just a quick follow-up for the record: I've recently updated my installed FFmpeg binaries, and with that, Texinfo was upgraded from version 7.1.1 to 7.2. It looks like this (minor) issue has been resolved upstream — the HTML documentation comments now correctly appear as:

```
<!-- Created by GNU Texinfo 7.2, https://www.gnu.org/software/texinfo/ -->
```

Glad to see it's been quietly fixed. No further action needed on this end.
